packages/net-security/sshguard/sshguard.exlib

```diff
@@ -15,31 +15,23 @@ LICENCES="
 "
 SLOT="0"
 
-MYOPTIONS="
-    ( providers:
-        hosts    [[ description = [ Use the hosts.allow/TCP wrappers backend ] ]]
-        iptables [[ description = [ Use the iptables/netfilter backend ] ]]
-    ) [[ number-selected = exactly-one ]]
-"
+MYOPTIONS=""
 
 DEPENDENCIES="
     build:
         sys-devel/bison
         sys-devel/flex
-    build+run:
-        providers:iptables? ( net-firewall/iptables )
+    recommendation:
+        net-firewall/iptables
     run:
         virtual/syslog
 "
 
-DEFAULT_SRC_CONFIGURE_OPTIONS=(
-    'providers:hosts --with-firewall=hosts'
-    'providers:iptables --with-firewall=iptables'
-    'providers:iptables --with-iptables=/usr/host/bin/iptables'
-)
-
 sshguard_src_install() {
     default
     install_systemd_files
+
+    insinto /etc
```

Comment:
I know there are probably existing examples, but I’d rather install the sample file to `/usr/share/sshguard` to keep `/etc` as clean as possible.